<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelAgencyUI</class>
 <widget class="QMainWindow" name="TravelAgencyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TravelAgencyUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="kundenGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Kundendaten</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Kunden-ID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Reisen:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="idLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTableWidget" name="ReisenTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Reise-ID</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Anfang der Reise </string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ende der Reise</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TravelD</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="reiseGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>Reise</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Reise-ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="reiseIdLineEdit">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Buchungen:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="BuchungenTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Buchungstyp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Start</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ende</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Preis</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionEinlesen"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <widget class="QMenu" name="menuSuche">
    <property name="title">
     <string>Suche</string>
    </property>
    <addaction name="actionKunden"/>
   </widget>
   <widget class="QMenu" name="menuBuchung">
    <property name="title">
     <string>Buchung</string>
    </property>
    <addaction name="actionHinzuf_gen"/>
    <addaction name="actionFehlermeldungen"/>
    <addaction name="actionAlle_checken"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSuche"/>
   <addaction name="menuBuchung"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEinlesen"/>
   <addaction name="actionKunden"/>
   <addaction name="actionSpeichern"/>
   <addaction name="actionHinzuf_gen"/>
   <addaction name="actionFehlermeldungen"/>
  </widget>
  <action name="actionEinlesen">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icon/open.png</normaloff>:/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Einlesen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionKunden">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icon/search.png</normaloff>:/icon/search.png</iconset>
   </property>
   <property name="text">
    <string>Kunden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icon/disk_718913.png</normaloff>:/icon/disk_718913.png</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHinzuf_gen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icon/add_11659161.png</normaloff>:/icon/add_11659161.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionFehlermeldungen">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icon/alerts.png</normaloff>:/icon/alerts.png</iconset>
   </property>
   <property name="text">
    <string>Fehlermeldungen</string>
   </property>
  </action>
  <action name="actionAlle_checken">
   <property name="text">
    <string>Alle checken</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
